sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/fragment/atoms/AtomsFragment.kt

```diff
@@ -17,47 +17,44 @@ package uk.gov.hmrc.sample_compose_fragments.presentation.fragment.atoms
 import androidx.navigation.fragment.findNavController
 import dagger.hilt.android.AndroidEntryPoint
 import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme
-import uk.gov.hmrc.components.compose.ui.theme.HmrcTheme.colors
 import uk.gov.hmrc.sample_compose_components.R
-import uk.gov.hmrc.sample_compose_components.databinding.FragmentAtomsBinding
+import uk.gov.hmrc.sample_compose_components.databinding.FragmentComposeExampleBinding
 import uk.gov.hmrc.sample_compose_fragments.data.repository.RepositoryImpl.Companion.ATOM_BUTTON
 import uk.gov.hmrc.sample_compose_fragments.data.repository.RepositoryImpl.Companion.ATOM_TEXT
 import uk.gov.hmrc.sample_compose_fragments.presentation.screens.ComponentListScreen
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.sampletemplate.HmrcSurface
 import uk.gov.hmrc.sample_compose_fragments.presentation.viewModel.AtomsViewModel
 
 @AndroidEntryPoint
-class AtomsFragment : Fragment(R.layout.fragment_atoms) {
+class AtomsFragment : Fragment(R.layout.fragment_compose_example) {
 
-    private lateinit var binding: FragmentAtomsBinding
+    private lateinit var binding: FragmentComposeExampleBinding
     private val viewModel: AtomsViewModel by activityViewModels()
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        binding = FragmentAtomsBinding.bind(view)
-        binding.composeViewAtoms.setContent {
-            val listItems by viewModel.atomsItems.collectAsState()
-            HmrcTheme {
-                Surface(
-                    modifier = Modifier.fillMaxHeight().fillMaxWidth(),
-                    color = colors.hmrcPageBackground
-                ) {
-                    ComponentListScreen(items = listItems, navigateTo = {
-                        when (it.id) {
-                            ATOM_TEXT -> findNavController().navigate(R.id.action_atomsFragment_to_textFragment)
-                            ATOM_BUTTON -> findNavController().navigate(R.id.action_atomsFragment_to_buttonFragment)
-                        }
-                    })
+        binding = FragmentComposeExampleBinding.bind(view)
+        binding.composeView.apply {
+            setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
+            setContent {
+                val listItems by viewModel.atomsItems.collectAsState()
+                HmrcTheme {
+                    HmrcSurface {
+                        ComponentListScreen(items = listItems, navigateTo = {
+                            when (it.id) {
+                                ATOM_TEXT -> findNavController().navigate(R.id.action_atomsFragment_to_textFragment)
+                                ATOM_BUTTON -> findNavController().navigate(R.id.action_atomsFragment_to_buttonFragment)
+                            }
+                        })
+                    }
                 }
             }
         }
```

Comment:
i have a question here in the sample we are just simply using xml only for ComposeView right?
instead of that we can simple use 

> ComposeView(requireContext()).apply { 
            setContent {}
        }

then we can get rid of layout xml files and binding ?